packages/upset/src/components/ElementView/ElementTable.tsx

```diff
@@ -40,10 +41,16 @@ function useColumns(columns: string[]) {
  * Table to display elements
  */
 export const ElementTable: FC = () => {
-  const attributeColumns = useRecoilValue(attributeAtom);
+  const attColumns = useRecoilValue(attributeAtom);
   const elements = useRecoilValue(selectedItemsSelector);
   const rows = useRows(elements);
-  const columns = useColumns(['_label', ...attributeColumns]);
+  const setColumns = useRecoilValue(setColumnsSelector);
+  let columns = useColumns(['_label', ...([...attColumns, ...setColumns].filter((col) => !col.startsWith('_')))]);
+  // Sort set columns to the right of other columns & add a boolean type to
```

Comment:
Does this sort? Looks like it just adds `type`

Reply:
The "sorting" here is really just the spread operator, putting the `setColumns` at the end of the list

Reply:
Actually, that comment is outdated; there used to be an explicit sort function but that was removed when I started using spread instead. Comment updated.